psi4/driver/procrouting/dft/hyb_functionals.py

```diff
@@ -646,6 +646,21 @@ def get_pw6b95_tweaks():
     '    BMK Hybrid Meta-GGA XC Functional for kinetics\n',
 })
 
+funcs.append({
+    "name": "t-HCTHh",
+    "x_functionals": {
+        "HYB_MGGA_X_TAU_HCTH": {
+            "use_libxc": True,
+        },
+    },
+    "c_functionals": {
+        "GGA_C_HYB_TAU_HCTH": {}
+    },
+    "description": '    Hybrid Tau HCTH Meta-GGA XC Functional\n',
+    "citation": '    A. D. Boese and N. C. Handy, J. Chem. Phys. 116, 9559, 2002\n',
+    "doi": "10.1063/1.1476309",
```

Comment:
I think it might be a good idea to tackle also this now: the description, citation and doi could be pulled from libxc...

Reply:
I appended doi to the citation. citation and description already pulled from libxc unless overwritten.